CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed: Allow pickup in Ghavoran Elun Transport Access by charging speed via navigation room
 - Changed: Help solver by adding Morph Ball requirment on connections to event to flip the spinner in Ghavoran Flipper Room
 - Changed: Shooting occluded objects requires at least Intermediate Knowledge
+- Changed: Connections in Cataris - Thermal Device Room North
```

Comment:
Please be more detailed on what changed

Reply:
Like this?

Reply:
Not really. You have a "Changed" entry that starts with "Fixed".

But other than that, try to read the entry from someone who comes entirely from outside, with no clue of what's going on. Could even be someone who's not familiar with the database.
You don't need to be detailed on how the change was made, but it should be understandable without checking the db.

In this case, something like "Fixed: Reorganised connections in ... to improve the solver".

Reply:
Gotcha will change it accordingly